AntennaTracker/AntennaTracker.cpp

```diff
@@ -137,6 +138,11 @@ void Tracker::ten_hz_logging_loop()
     }
 }
 
+void Tracker::update_adsb()
```

Comment:
when using SCHED_TASK_CLASS, you won't need this at all. the scheduler calls adsb.update() directly